use-hot-module-reload/package.json

```diff
@@ -2,50 +2,48 @@
   "name": "use-hot-module-reload",
   "version": "1.0.3",
   "description": "React hook that triggers a callback after hot-module reloading has been performed (for any module, not just the one it was used in)",
+  "keywords": [
+    "hmr",
+    "hot-module-reload",
+    "webpack-vite"
+  ],
+  "homepage": "https://github.com/rexxars/use-hot-module-reload#readme",
+  "bugs": {
+    "url": "https://github.com/rexxars/use-hot-module-reload/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/rexxars/use-hot-module-reload.git"
+  },
+  "license": "MIT",
+  "author": "Espen Hovlandsdal <espen@hovlandsdal.com>",
+  "sideEffects": false,
+  "type": "module",
   "exports": {
     ".": {
-      "require": "./dist/use-hot-module-reload.cjs.js",
-      "node": {
-        "import": "./dist/use-hot-module-reload.esm.mjs",
-        "require": "./dist/use-hot-module-reload.cjs.js"
-      },
```

Comment:
These are not needed since they resolve to the same files as the upper level `require` and `import` conditions